openapi/openapi/src/main/scala/endpoints/openapi/JsonSchemas.scala

```diff
@@ -81,6 +81,8 @@ trait JsonSchemas extends endpoints.algebra.JsonSchemas {
 
   def invmapJsonSchema[A, B](jsonSchema: DocumentedJsonSchema, f: A => B, g: B => A): DocumentedJsonSchema = jsonSchema
 
+  lazy val uuidJsonSchema: DocumentedJsonSchema = Primitive("uuid")
```

Comment:
I think it should be `Primitive("string", format = Some("uuid"))` instead?

Reply:
Ah, yes; thanks.